frontend/device/sdl/device.lua

```diff
@@ -34,10 +34,22 @@ function Device:init()
     end
 
     if util.haveSDL2() then
+        self.hasClipboard = yes
```

Comment:
If will work out if the box for android right? There's no need to add same entry in android device.lua right?

Reply:
It will work out of the box for android right? There's no need to add same entry in android device.lua right?

Reply:
That depends on what you mean by out of the box. Wrt to this PR Android would default to `no`. This is only for the SDL2 abstraction. I haven't added it to Android because I couldn't test it or anything. :-)

Reply:
@mwoz123 I added a commit with Android definitions but I might still decide to cherry-pick that into another PR.